components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -418,6 +437,20 @@ private static void setReCaptchaConfigs(Properties properties) {
         }
         CaptchaDataHolder.getInstance().setReCaptchaRequestWrapUrls(reCaptchaRequestWrapUrls);
 
+        String reCaptchaScoreThreshold = properties.getProperty(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD);
+        if (StringUtils.isBlank(reCaptchaScoreThreshold)) {
+            if (log.isDebugEnabled()) {
+                log.debug("Error parsing recaptcha.threshold from config. Hence using the default value : " +
+                        CaptchaConstants.CAPTCHA_V3_DEFAULT_THRESHOLD);
+            }
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(CaptchaConstants.CAPTCHA_V3_DEFAULT_THRESHOLD);
+        }
+        try {
+            CaptchaDataHolder.getInstance().setReCaptchaScoreThreshold(Double.parseDouble(reCaptchaScoreThreshold));
+        } catch (NumberFormatException e) {
+            throw new RuntimeException(getValidationErrorMessage(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD));
```

Comment:
`getValidationErrorMessage(CaptchaConstants.RE_CAPTCHA_SCORE_THRESHOLD)` what this method returns?

Reply:
`"Invalid value for " + property + " in the " + CaptchaConstants  .CAPTCHA_CONFIG_FILE_NAME + " file."`